packages/cli/src/db-studio/api/http/router.ts

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
files will need copyright header; IDEA will apply it for you automatically, can help configure it in discord